service/cron.py

```diff
@@ -1,5 +1,5 @@
 from datetime import datetime
-from .models import Massage
+from .models import Massage, Schedule as Sch
```

Comment:
Why do you need Sch? Why can you not use Schedule?
Please, use more clearly names

Reply:
Done